lib/rails_upgrade_helpers/version.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module RailsUpgradeHelpers
+  class Version
+    class << self
+      def if_version(rails_4: nil, rails_5: nil, rails_6: nil)
+        case Rails::VERSION::MAJOR
```

Comment:
We need to be careful with this.

`aggregate` doesn't have a hard dependency on `Rails`..it only has one on `ActiveRecord`.

You're getting lucky here because `aggregate` includes `large_text_field` which has a `Rails` dependency...but its a bad practice to directly reach into some module/gem if it isn't a direct dependency.

You can determine that be looking at the [gemspec](https://github.com/Invoca/aggregate/blob/master/aggregate.gemspec#L22) and [Gemfile](https://github.com/Invoca/aggregate/blob/master/Gemfile) files. Nowhere is `rails` determined to be an actual dependency, only `activerecord`

This file should be revamped to be based off of `ActiveRecord` version as a result